ui/src/app/base/components/Col/Col.md

```diff
@@ -0,0 +1,160 @@
+## Grid
+
+```jsx
```

Comment:
Would the rendered examples be clearer here with borders around the cells?

Reply:
Done, I've made this match the Vanilla docs: https://docs.vanillaframework.io/patterns/grid/.